tslint.json

```diff
@@ -32,6 +32,7 @@
     "prettier": true,
     "saga-yield-return-type": true,
     "no-tslint-disable-all": true,
+    "no-big-function": false,
```

Comment:
use `tslint:disable-next-line:no-big-function` instead of disabling it globally

Reply:
I did a push with the requested change. I attached also the requested screenshots.

![Screenshot_1554996406](https://user-images.githubusercontent.com/10615049/55970334-a4620500-5c7f-11e9-9ea7-8cd2840aa3a1.png)
![Simulator Screen Shot - iPhone X - 2019-04-11 at 17 26 05](https://user-images.githubusercontent.com/10615049/55970335-a4620500-5c7f-11e9-9b7f-30e2a724856d.png)